src/mod/applications/mod_conference/conference_loop.c

```diff
@@ -823,8 +823,6 @@ void *SWITCH_THREAD_FUNC conference_loop_input(switch_thread_t *thread, void *ob
 
 	channel = switch_core_session_get_channel(session);
 
-	switch_core_session_get_read_impl(session, &member->read_impl);
```

Comment:
why you remove this line from here?

Reply:
`member->read_impl` is populated by the call `conference_loop_output()` before the thread starts, so calling it again here is redundant.

Reply:
Yes, as far as I can tell the `read_impl` does not change after the input/output loops are started. Am I missing something?

Reply:
https://github.com/signalwire/freeswitch/blob/master/src/mod/applications/mod_conference/mod_conference.c#L2489

Reply:
I see, there's another path for spawning the input loop, thanks for catching that. I've rebased and updated the patch to not drop the initialisation in the input loop.

Reply:
ty